Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml

```diff
@@ -8,6 +8,9 @@
     sprite: Objects/Weapons/Melee/pickaxe.rsi
     state: pickaxe
   - type: Pickaxe
+    damage:
+      types:
+        Piercing: 25
```

Comment:
How long is the do-after for an average rock? Could a user deal more damage by just attacking the rock rather than mining?

If that's the case, maybe the `MeleeWeaponComponent`'s `cooldownTime` & `arcCooldownTime` need to be longer

Reply:
mining is faster in this case, unless you're using wide attacks i guess?

Reply:
will increase the arc cooldown time